tracer/build/_build/Build.cs

```diff
@@ -320,7 +320,8 @@ void DeleteReparsePoints(string path)
                 (rid: "linux-x64", archiveFormat: ".tar.gz"),  
                 (rid: "linux-musl-x64", archiveFormat: ".tar.gz"),  
                 (rid: "osx-x64", archiveFormat: ".tar.gz"),  
-                (rid: "linux-arm64", archiveFormat: ".tar.gz") 
+                (rid: "linux-arm64", archiveFormat: ".tar.gz"),
+                (rid: "osx-arm64", archiveFormat: ".tar.gz")
```

Comment:
There's a _lot_ more places you need to add osx-arm64 when we want to build in CI, but I think for now you should probably update:

- tracer/test/Datadog.Trace.TestHelpers/EnvironmentHelper
- tracer/src/Datadog.Trace.Tools.Runner/Datadog.Trace.Tools.Runner.csproj
- shared/src/Datadog.Trace.ClrProfiler.Native/loader.conf
- tracer/src/Datadog.Trace.Tools.Runner/Utils.cs

Reply:
true!, done in [01afc7d](https://github.com/DataDog/dd-trace-dotnet/pull/3279/commits/01afc7d32024a6404924d48ebc3eccff372fbe42)